012-infinispan-secure-server/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Use `013-` prefix, Pablo uses 012 in his PR which arrived earlier

Reply:
ok, got you